src/routes/notification/types/index.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import NotificationTypeController from '../../../controllers/NotificationTypeController';
+
+const router = express.Router();
+
+router.get('/', NotificationTypeController.getTypes);
+router.get('/:id', NotificationTypeController.getTypes);
+router.post('/', NotificationTypeController.createType);
```

Comment:
The post, put, and delete routes should be in a separate auth/index.js file so they can be registered under a different path like the User routes are.

Reply:
done